sec-keys/openpgp-keys-apache-jackrabbit/openpgp-keys-apache-jackrabbit-20250728.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+SEC_KEYS_VALIDPGPKEYS=(
+	'CBA5A7C21EC143314C41393E5B968010E04F9A89:fielding:manual'
+	'402F685604A7EFD6FA2BDFD850634F398A26D9A6:zitting:manual'
+	'EAA526B91DD83BA3E1B9636FA730529CA355A63E:zitting:manual'
+	'369AF551BA81A412C3D413675E27F86EF79B7715:meschberger:manual'
+	'2289AC4180E130507D7A82F479C211E0AEFCA72E:parvulescu:manual'
+	'D61B5BA9EE35A22B1DF12D799378564DAE0B5181:schrijvers:manual'
+	'2B5424CD1B4A050A44AAECB4948C7604A23CB531:schans:manual'
+	'B99405433FA2E3611F1A1669049B39036301DF93:schans:manual'
+	'35DC0B0F3063D253D677603FB39DC973EB2F0489:reschke:manual'
+	'08BA46AB3C7130EBAE66DED39AC2DE9E1BB83D7C:baedke:manual'
+	'8D646491C0E5E287CD94891C2D0D21B66D071E25:bocanegra:manual'
+	'BE0E717C463E0EBA6E5005C2D108FF879AA63950:reutegger:manual'
+	'ED09733FA05DEDF15100ABEB32BBF14AF07CA77B:graf:manual'
+	'C8B293879EA9CB39C1A1ADB49E5A18C3AEA9D105:duerig:manual'
+	'BCD0F74396EF7B7735092F8B180BA1CD74628A7F:jain:manual'
+	'0012E1169C9CD53EEC0C7185C6D825C35A51AC9D:giannella:manual'
+	'37764359E96FDCB167611DD1F3DE8E1B88E59E02:mehrotra:manual'
+	'0E61C201E0E9A16997B3DCA096B413CA807BA9CB:jaggi:manual'
+	'FD4448FC1C8FACC0878F72FD5CDB3134085450C5:jain:manual'
+	'B855B0AEE8957FE46ACC322A576C34605AD04097:mari:manual'
+	'44F4797A52C336FA666CD9271DE461528F1F1B2A:reschke:manual'
+	'91BD884E10B0D83A0AEB1BF4CEE01CFADD5F338B:saurabh:manual'
+	'0B616D26686A93F18089869EFEFFEA35770E6354:baedke:manual'
+	'04DE676E3FFFD9C2DD767C2FA25D65D27C13ADCE:ko:manual'
+	'FCC8B6BC39942279732439CABFD436525B570D3A:angela:manual'
+	'3C744F7A9E6208E2079C894678362BD4CC5B2D91:gupta:manual'
+	'B91AB7D2121DC6B0A61AA182D7742D58455ECC7C:windszus:manual'
+	'2817F597FE5251439F41704011B1A21AD89AFDED:dulceanu:manual'
+	'A3322645D37AD83BAEFF3991CEF68B0987305F6D:ngupta:manual'
+	'FF76B506F48C17F46AFEB9F0BB2EAB31A732F339:smiljanic:manual'
+	'07D03FA3C72D170CC0456BD530898A86ABE8F08A:kumar:manual'
+	'C536C8CAA12C0CFC0DF840367D27D8A059FE68E2:hoh:manual'
+)
```

Comment:
There is only one key you really need. You will get it by running
`gpg /var/cache/distfiles/jackrabbit-2.20.27-src.zip.asc`

For a newer example pls compare https://github.com/gentoo/gentoo/pull/44512/commits/fcf3090b16fc6fb8ea24d18248b0d7908df01924.